client/app/components/pas-form.js

```diff
@@ -5,6 +5,11 @@ import { tracked } from '@glimmer/tracking';
 export default class PasFormComponent extends Component {
   @tracked package;
 
+  @action
+  save(packageInstance) {
+    packageInstance.saveDescendants();  
```

Comment:
Thinking about this some more... this makes our pas-form component class and our package model turn into an example of [inappropriate intimacy](https://refactoring.guru/smells/inappropriate-intimacy).

This is fine for now but just some food for thought later down the road :)

Reply:
One way to deal with this is to pass down a generic "onSave" handler. The Pas-Form component could simply call 'onSave' when the button is clicked, and fire the passed handler. It could be defined as a noop by default.

Reply:
@allthesignals totally agree with you on this -- I added a TODO comment for now.